src/chrome/content/rules/Better-Business-Bureau.xml

```diff
@@ -1,4 +1,8 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://hurdman.app.bbb.org/ => https://hurdman.app.bbb.org/: (6, 'Could not resolve host: hurdman.bbb.org')
+
```

Comment:
hurdman.app.bbb.org/$ redirects to the same non-existent host whether over http or https, so this isn't a problem.